src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderChooser.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {FC, useContext} from 'react'
+import {
+  AlignItems,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  InputLabel,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Style
+import './ProviderChooser.scss'
+
+// Components
+import {CreateOrgContext} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext'
+import {ClusterBox} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ClusterBox'
+import {RegionDropdown} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/RegionDropdown'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+export const ProviderChooser: FC = () => {
+  const {clusters} = useContext(CreateOrgContext)
+  const clusterKeys = Object.keys(clusters)
+
+  return (
+    <FlexBox
+      alignItems={AlignItems.FlexStart}
+      className="provider-and-region-chooser"
+      direction={FlexDirection.Column}
+      justifyContent={JustifyContent.Center}
+      margin={ComponentSize.Large}
+      stretchToFitWidth={true}
+    >
+      <InputLabel className="prc-title">Provider & Region</InputLabel>
+      <InputLabel className="prc-description" size={ComponentSize.Small}>
+        Tell us where you would like to store the time series data for this
+        organization.
+      </InputLabel>
```

Comment:
I'm not sure this is a proper use of `InputLabel`. According to the [storyboard](https://influxdata.github.io/clockface/?path=/story/components-inputs-base--inputlabel)

>InputLabel is designed to visually and functionally label an adjacent input element.

An HTML label is meant to label a form element (like an `input` or `checkbox`) and is usually used in the context of a form. I think semantically, it might be more appropriate to use a `Heading` element (so like `h3` or `h4`)